reset-pass/account-validator-tool/src/webapp/css/newUser.css

```diff
@@ -35,6 +35,26 @@ p, h1, form, button {
 	width:50em;
 }
 
+.stylized .topheading {
+    text-align: center;
+    margin-bottom: 12px;
+}
+
+.stylized .logo {
+    background: #444 url(/library/skin/morpheus-default/images/logo.png) no-repeat center center;
```

Comment:
Sam, our feeling is that what we've done improves things greatly. Admittedly there's a better solution but this would take a lot of time that we dont have. I'm hoping that this can be accepted along with a new jira which specifies exactly what the full solution would be